CHANGELOG.md

```diff
@@ -116,12 +116,12 @@ particular we'd like to bring to your attention first.
      before `v0.25.0` will start. Auto-update notifications for Windows will
      resume on the
      next release. See the
-     [Installation](https://github.com/brimdata/brim/wiki/Installation#windows-installation-v0250)
+     [Installation](https://github.com/brimdata/brim/blob/v0.25.0/docs/Installation.md#windows-installation-v0250)
```

Comment:
For many of these old wiki docs referenced from the changelog, I've changed them to point to tagged references in the `docs/` directory. One of the good/bad properties of the GitHub wiki is that links to pages that don't exist end up being redirected to the top-level wiki home. That's arguably better than a 404, but if there was any reason for a user to actually reference some old doc because it's mentioned in the changelog, I figure actually pointing them at it is preferable to starting them at the home page and them not seeing the doc. For docs that still live at their original location referenced in the changelog, I've left those links as is.